auth0/resource_auth0_client_test.go

```diff
@@ -88,7 +93,7 @@ resource "auth0_client" "my_client" {
   is_first_party = true
   is_token_endpoint_ip_header_trusted = true
   token_endpoint_auth_method = "client_secret_post"
-  oidc_conformant = false
+  oidc_conformant = true
```

Comment:
Can we revert this? I think CI might break with this set to false due to the tenant configuration...

Reply:
can't really revert, i think. When it's set to false, I get this during the testacc run:
"Error: 400 Bad Request: Application must be OIDC Conformant when Refresh Token rotation is enabled"